packages/extension/src/view/devtools/dashboardApp.tsx

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
These dashboard files should be moved to a new folder, probably `report` under `view/devtools`